matcher/matcher_config.yaml

```diff
@@ -44,38 +44,8 @@ contrasts:
                method: jarowinkler
     middle_name:
         - method: compare_exact 
-        - method: compare_exact 
-          args:
-               n_chars: 5
-        - method: compare_exact 
-          args:
-               n_chars: -5
-        - method: compare_exact 
-          args:
-               n_chars: 3
-        - method: compare_exact 
-          args:
-               n_chars: -3
-        - method: compare_string_distance
           args:
-               method: jarowinkler
-    last_name:
```

Comment:
Did you mean to remove the last name section as well? The PR description just mentions the middle name